### PR TITLE
Contribute mod

### PR DESCRIPTION
**Pull request by zkamvar**
I have made modifications to CONTRIBUTING.md from @smanel's initial changes to remove the word "workflow" (addressing #54). 

I have addressed issues #39, #59, #60, and #25. 
I have also updated the template and changed the name from example_contrib.Rmd to TEMPLATE.Rmd.

### CODE REVIEW

**Comment by zkamvar** [comment]
@coughls, @smanel, and @hlapp, can you check the files and see if they are okay to merge?

**Comment by coughls** [comment]
I added one suggested comment to a vignette but its only a suggested change so am happy to merge anyway. @smanel, @hlapp @zkamvar

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,36 +1,56 @@
...
-2. Fork the [source repository](http://github.com/NESCent/popgenInfo) on Github using the `Fork` button. 
-3. Clone the repository to your machine using `git clone https://github.com/NESCent/popgenInfo.git` on the command line. The URL used here is in the HTTPS clone url section on the popgenInfo github page. Keep your clone synced up by following Step 3 of [this tutorial](https://help.github.com/articles/fork-a-repo/). 
-4. Then, move to the popgenInfo directory using `cd popgenInfo`.
+# How do I submit a vignette?
+
+Overall, vignette submission will follow the guidelines in [GitHub flow](https://guides.github.com/introduction/flow/index.html). Please create a new branch for each vignette to be submitted. 
+
+ 
+ 
+## Step-by-step Process for submission
```
Comment (hlapp) [inline CONTRIBUTING.md]: Consistent capitalization? I.e., _Process_ should probably be lower case, right?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,36 +1,56 @@
...
+
+1. Set up a GitHub account if you don't already have one, see [here](https://guides.github.com/activities/hello-world/) for a quick tutorial on using GitHub. Download an appropriate Git client for your computer, see [here](https://help.github.com/articles/set-up-git/), and set it up.
+2. Fork the [source repository](http://github.com/NESCent/popgenInfo) on GitHub using the `Fork` button. 
+3. Clone the repository to your machine using `git clone https://github.com/YourAccount/popgenInfo.git` on the command line. The URL used here is in the HTTPS clone url section on the popgenInfo GitHub page. Keep your clone synced up by following Step 3 of [this tutorial](https://help.github.com/articles/fork-a-repo/).
```
Comment (hlapp) [inline CONTRIBUTING.md]: I think we should indicate here as a short note that the HTTPS URL will be easier to follow for beginners (no need for setting up an ssh key), but that the SSH URL will work just as well (and in fact better if one has an SSH key). So I'm wondering whether we can't just refer to [Github's help](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork).

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,36 +1,56 @@
...
+ - Vignette must be able to successfully run examples with available packages and data (this is automatically checked)
+ - At least two people other than the vignette writer must approve the pull request
+
+## Best practices
+
+ - Use the [template](https://github.com/nescent/popgenInfo/tree/master/TEMPLATE.Rmd) to get started
+ - Give your vignette a unique name by using the date of your first commit (e.g. `2015-06-16_Parentage_Microsatellite.Rmd`) would be a vignette that was started on June 16th, 2015) 
+ - List authors at the bottom of the vignette
+ - Use small examples and encourage readers to explore with bigger examples
+ - Submit your vignette early to get feedback
```
Comment (hlapp) [inline CONTRIBUTING.md]: Add a statement to indicate on the pull request description if the submission isn't considered ready for merging but primarily there for obtaining feedback?

File: TEMPLATE.Rmd [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+## Section 1: Load the data
+
+
+```{r, load_data_show, eval=FALSE}
+nancycats <- import2genind("nancycats.gtx")
+summary(nancycats)
+```
+
+```{r, load_data_evalutate, echo=FALSE}
```
Comment (hlapp) [inline TEMPLATE.Rmd]: Typo in chunk label.

File: CONTRIBUTING.md (hunk 1 above)
Comment (zkamvar) [inline CONTRIBUTING.md]: correct

**Comment by zkamvar** [comment]
I've addressed @hlapp's comments and can change @coughls's suggestion if @smanel is okay with it.

**Pull request merged**